package.json

```diff
@@ -24,5 +24,10 @@
     "workspaces": [
         "apps/*",
         "packages/*"
-    ]
+    ],
+    "dependencies": {
+        "@elastic/eui": "^97.0.0",
+        "@testing-library/react": "^16.0.1",
+        "react-redux": "^8.1.3"
```

Comment:
As far as i know these do not belong here, since this is the package.json of the mono repo itself. Such dependencies would be more an app related dependency. `Discussion`